src/constants.js

```diff
@@ -273,7 +273,7 @@ export const AVATAR = {
 	SIZE: {
 		EXTRA_SMALL: 22,
 		SMALL: 32,
-		DEFAULT: 36,
+		DEFAULT: 39,
```

Comment:
Is it actually the "default" size? It seems that it is the size of avatar in list items, defined by the list item height (= conversations list and participants list).

Reply:
which is also most widely used in project

Reply:
But this size is 39 not because we want the default size to be 39 wherever it's used, but because we want it exactly in the list items